pdns/tcpreceiver.cc

```diff
@@ -823,15 +823,24 @@ int TCPNameserver::doAXFR(const DNSName &target, std::unique_ptr<DNSPacket>& q,
     }
     zrr.dr.d_name.makeUsLowerCase();
     if(zrr.dr.d_name.isPartOf(target)) {
-      if (zrr.dr.d_type == QType::ALIAS && ::arg().mustDo("outgoing-axfr-expand-alias")) {
+      if (zrr.dr.d_type == QType::ALIAS && (::arg().mustDo("outgoing-axfr-expand-alias") || ::arg()["outgoing-axfr-expand-alias"] == "ignore-errors")) {
```

Comment:
`mustDo` considers `ignore-errors` as a positive answer already. However, if we ever change that, it would be good if this code survives it, so you can leave it in.

Reply:
Can't say I expected mustDo to consider this a positive answer in the first place :)

Reply:
`notNoOrOff` would be a more accurate name ;)

Reply:
right :)